lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -105,6 +105,13 @@ def extract_method(node)
           method_name
         end
 
+        # Only add the method to the chain if the method being
+        # called if it's a member of the time class.
+        def add_to_chain(node)
```

Comment:
In general - the name of this method sounds confusing. The relationship between this check and the method chain concept in just marginal at best.